tests/ctst/steps/utils/kubernetes.ts

```diff
@@ -25,6 +26,25 @@ type ZenkoStatusValue = {
 
 type ZenkoStatus = ZenkoStatusValue[];
 
+type ZenkoVersion = {
+    apiVersion: string;
+    kind: string;
+    metadata: {
+        name: string;
+    };
+    spec: ZenkoVersionSpec;
+};
+
+type ZenkoVersionSpec = {
+    versions: {
+        s3utils: {
```

Comment:
also pick up the "image" to use

```suggestion
        s3utils: {
            image: string;
```